src/command/use.rs

```diff
@@ -0,0 +1,29 @@
+use structopt::StructOpt;
+
+use crate::command::Command;
+use notion_core::session::{ActivityKind, Session};
+use notion_fail::{ExitCode, Fallible};
+
+#[macro_export]
+macro_rules! usage {
+    () => {
+        "notion-use
+
+DEPRECATED:
+    To install a tool in your user toolchain, use `notion install`.
```

Comment:
This text should also be updated, similar to the top-level help text.

Reply:
Done!